api/pkg/apis/v1alpha1/utils/parser.go

```diff
@@ -1456,12 +1456,14 @@ func enumerateProperties(js interface{}, context utils.EvaluationContext) (inter
 				parser := NewParser(strVal)
 				val, err := parser.Eval(context)
 				if err != nil {
+					log.ErrorfCtx(context.Context, " (Parser): Enumerate properties failed: %v", err)
```

Comment:
I believe parser has lots of error return but not logged. Do we choose these three places to expose the error?

Reply:
add logging in parser.Eval()